### ISSUE TITLE
PeoplePicker: Cannot navigate through People Picker suggestions on Edge with Narrator ON

### ISSUE THREAD

**Issue by ssingh28** [OfficeDev/office-ui-fabric-react#5286]
This issue can be observed even on the MS Fabric page https://developer.microsoft.com/en-us/fabric#/components/peoplepicker  on Edge with Narrator ON

The only way people picker works is if the control comes via mouse rather than keyboard. Also once the item is chosen if backspace/delete is clicked Narrator reads out "unavailable". This same thing is read out when someone tries to type a name is the control. Attached is the video of the same.

**Comment by JasonGore** [comment]
I seem to be able to navigate across items using keyboard navigation now. There are a variety of other issues but they seem pretty well covered by #6259. I will close this issue as a duplicate.
